components/Footer/Footer.tsx

```diff
@@ -13,22 +13,35 @@ export default function Footer() {
             <h3>{t('footer.getinvolved')}</h3>
             <ul>
               <li>
-                <Link href={`/${locale}/community#event-formats`}>
+                <Link
+                  href={`/[lang]/community#event-formats`}
+                  as={`/${locale}/community#event-formats`}
```

Comment:
That's interesting, i didn't know that we can just take url params without actually extracting them.
But do we actually need to add the `as`? That's for when a pretty url is needed but our url is already the same, i mean if we use `lang` or `locale` it always resolute in i.e. `en/community#...`

Reply:
I guess you have to configure the directory name somewhere. It's a bit annoying to do it every time. But otherwise it's broken and it's clearly documented how to do it.

Maybe we could extract a `<LangLink>` component or something.

Reply:
ok, yes we need to use `as` for dynamic route as discussed. 
Let's start to merge like that and if it become tedious we can think about a `LangLink` component, what do you think?